Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -2,6 +2,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection.Metadata.Ecma335;
```

Comment:
What is this namespace for? 😕

Reply:
LOL, I honestly have no idea? Might have gotten imported somehow.